SeaPublicWebsite/Constants.cs

```diff
@@ -2,5 +2,6 @@
 
 public static class Constants
 {
+    public const string FEEDBACK_URL = "https://sea-beta-dev.cloudapps.digital/feedback";
```

Comment:
This doesn't work on prod, but we're about to change feedback to be a separate site so then one link will make sense.